storyblok/storyblok-design-system
# Task(on-324): Activate new tab link on the sidebar

### This PR adds the possibility to open the sidebar links into a new window



## Pull request type

Jira Link: [ON-324](https://storyblok.atlassian.net/browse/ON-324)




## How to test this PR

- Open a space (it has more links on side bar)
- Right-click with mouse on a sidebar link and see that you have an option to open that in a new window

## Other information

This PR removes href binding for router-link because it was overriding router-link generated value as soon as we click on a sidebar list item
![Captura de ecrã 2022-03-29, às 10 51 08](https://user-images.githubusercontent.com/94900845/160585009-d4bb1bea-6eb4-4e83-8d5e-df7241c918bb.png)

Video:

https://user-images.githubusercontent.com/94900845/160793594-526dc52a-346c-478f-a596-a721b6b212b7.mov
